### PR TITLE
Dont show in app notification for own send message

### LINKED ISSUE #315: Regarding snackbar notification
1. When snackbar is shown and on clicking **Show** button user id narrowed, and then again coming to home user can not find that new message at the end of the list. On the other hand if snackbar is auto hidden new message is added to list.
videotogif_2017 01 21_05 10 45

2. On clicking **Show** button user is narrowed, it will be better to scroll to the latest(new) messages.
videotogif_2017 01 21_05 15 50

2. As of now snackbar is show even if we are sending message. I don't think snackbar notification is required for own send messages.
videotogif_2017 01 21_05 12 48

Please review and let me know, this are bugs or intentionally done like this. I would love to work on this.

### PR DESCRIPTION
**Pull request by vishwesh3**
Fix #315 3rd one

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@vishwesh3, it looks like you've already signed the Dropbox CLA. Thanks!

File: app/src/main/java/com/zulip/android/activities/ZulipActivity.java [hunk 1]
```diff
@@ -2148,8 +2148,8 @@ private void showSnackbarNotification(Message[] messages) {
         MutedTopics mutedTopics = MutedTopics.get();
         int nonMutedMessagesCount = 0;
         Message tempMessage = null; //Stores a temporary message which is non-muted, later used for retrieving Stream/Topic
-        for (Message message : messages) { //Check if all messages from same topic/private and remove all the muted messages
-            if (mutedTopics.isTopicMute(message) || ((message.getStream() != null) && !message.getStream().getInHomeView())) {
+        for (Message message : messages) { //Check if all messages from same topic/private and remove all the muted messages or send by user itself
+            if (mutedTopics.isTopicMute(message) || ((message.getStream() != null) && !message.getStream().getInHomeView()) || message.getSender().getEmail().equals(app.getEmail())) {
```
Comment (kunall17) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: `app.getEmail()` also works but use app.getYou().getEmail()

Comment (vishwesh3) [inline app/src/main/java/com/zulip/android/activities/ZulipActivity.java]: Done.

**Comment by kunall17** [comment]
@timabbott LGTM!

**Comment by timabbott** [comment]
Merged as 6afb94c3e120e001afddef8ba0f0dd4a46e08e6f, thanks @vishwesh3!  I just added a note that this fixes part 3 of #315 in the commit message.

**Pull request closed without merging**